NetCoreRestApi/Common/Converter/IConverter.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace Common.Converter
+{
+    public interface IConverter<T1, T2>
+    {
+        T1 ConvertTo(T2 t2);
+        T2 ConvertFrom(T1 t1);
```

Comment:
Replace T1 with T2 and vice versa. we should understand that we convert T1 to T2

Reply:
I need help with that

Reply:
done